### PR TITLE
Disable rename and copy menus

### PR DESCRIPTION
**Pull request by martinRenou**
cc. @fcollonval

Hiding the cut/copy/rename context menu commands.

### CODE REVIEW

**Comment by martinRenou** [comment]
I have some issues with this PR:

1) It seems to work almost correctly but the rename/copy/cut menus are added back on top and do not respect the rank on the classic file browser:

cut

2) Also it's a bit annoying because this approach does not work with jupyterlab-unfold unless we use the same id.

3) I'm unsure how to install the `overrides.json` file, should this be done manually in the `setup.py`? Or is there a custom entry to add in the `package.json` (like it is done for the schema directory)?

**Comment by martinRenou** [comment]
> 1. It seems to work almost correctly but the rename/copy/cut menus are added back on top and do not respect the rank on the classic file browser:

This should be fixed by [jupyterlab/jupyterlab#10666](https://github.com/jupyterlab/jupyterlab/issues/10666)

> 2. Also it's a bit annoying because this approach does not work with jupyterlab-unfold unless we use the same id.

I'll see if I can rework this

**Comment by fcollonval** [comment]
> I have some issues with this PR:
> 
>     1. It seems to work almost correctly but the rename/copy/cut menus are added back on top and do not respect the rank on the classic file browser:

That PR https://github.com/jupyterlab/jupyterlab/pull/10666 will need to be backported to resolve that.
 
>     2. Also it's a bit annoying because this approach does not work with jupyterlab-unfold unless we use the same id.

Yes this is where things start to be complex.

A possibility is to deactivate the entry for the core command. And to create a new command that does exactly the same but with a `isVisible` method that returns false for this widget and true otherwise.

>     3. I'm unsure how to install the `overrides.json` file, should this be done manually in the `setup.py`? Or is there a custom entry to add in the `package.json` (like it is done for the schema directory)?

You don't need to add the `overrides.json`, you can place every entries in this extension `jupyter.lab.menus.context` section of the plugin schema.

**Comment by martinRenou** [comment]
Thanks for your help @fcollonval! 

Now only remains the issue that https://github.com/jupyterlab/jupyterlab/pull/10666 should fix. I don't have a strong opinion whether we should wait for the fix to be released or not. It's a trade-off between having menus that don't work versus misplaced menus.

https://user-images.githubusercontent.com/21197331/163342020-2c8d7417-d6be-4736-86c7-06b92c14c98f.mp4

File: src/index.ts [hunk 1]
```diff
@@ -49,6 +57,9 @@ const plugin: JupyterFrontEndPlugin<void> = {
     widget.title.caption = trans.__('Local File System');
     widget.title.icon = listIcon;
 
+    // Adding a data attribute
+    widget.node.setAttribute('data-is-filebrowser-access', '');
```
Comment (jtpio) [inline src/index.ts:61]: Maybe this should be?

```suggestion
    widget.node.setAttribute('data-is-filesystem-access', '');
```

**Comment by jtpio** [comment]
Thanks @martinRenou!

Looks like the manifest check is failing because of the `schema` folder.

**Comment by jtpio** [comment]
Thanks!

**Pull request merged**